### ISSUE TITLE
3.1.0 Package implementation breaks app usage.

### ISSUE THREAD

**Issue by jetpeter** [FlutterGen/flutter_gen#102]
In the latest version application assets are pathed through a package directory that does not exists. This can be observed in the example project. 
```  
SvgPicture svg({
    Key? key,
    bool matchTextDirection = false,
    AssetBundle? bundle,
    String? package = 'example',
...
```
In this case the example package does not exist in the example project so the following excepion occurs.
```
======== Exception caught by image resource service ================================================
The following assertion was thrown resolving an image codec:
Unable to load asset: packages/example/assets/images/chip1.jpg

When the exception was thrown, this was the stack: 
#0      PlatformAssetBundle.load (package:flutter/src/services/asset_bundle.dart:227:7)
<asynchronous suspension>
#1      AssetBundleImageProvider._loadAsync (package:flutter/src/painting/image_provider.dart:667:14)
<asynchronous suspension>
Image provider: AssetGenImage(bundle: null, name: "packages/example/assets/images/chip1.jpg")
Image key: AssetBundleImageKey(bundle: PlatformAssetBundle#732c2(), name: "packages/example/assets/images/chip1.jpg", scale: 1.0)
```

The only way to fix this is to manually set package to null in asset widget:
`Assets.images.icons.fuchsia.svg(package: null)`

Is there a way to blanket disable the package name in version 3.1.0 for all assets?

**Comment by lcdsmao** [comment]
Thank you for the report. I can reproduce this bug.
We will disable this feature by default and add an option to enable it in the next release.

**Comment by britannio** [comment]
Was this issue present in the example app? Fortunately I didn't waste much time tracking this issue down with the help of [git-bisect](https://git-scm.com/docs/git-bisect) but if such issues could be caught via a hybrid of [integration_test](https://flutter.dev/docs/testing/integration-tests) and [golden_toolkit](https://pub.dev/packages/golden_toolkit) in the future, this package would be more robust. 

Thanks for the quick fix anyway!

**Comment by wasabeef** [comment]
@jetpeter 
Thank you for your issues.
We just released the fixed version, please upgrade to `v3.1.1`.

@britannio 

Nice. We’ll look into it.
